app/controllers/miq_ae_class_controller.rb

```diff
@@ -197,6 +198,13 @@ def get_node_info(node)
     x_history_add_item(:id => x_node, :text => @right_cell_text)
   end
 
+  def add_domain_version_message
+    version = @record.version
+    available_version = @record.available_version
+    return if version.nil? || available_version.nil?
+    @version_message = _("Currently running %s version, you have %s version available.") % [version, available_version] if version != available_version
```

Comment:
@gmcculloug about your other comment about showing this message on Datastore node, i had initially thought that it should only be displayed on Root node of Automate explorer tree, but model methods that are added are instance methods, @mkanoor suggested to add message on Domain level, i can rework these changes if model method can be changed to return version info for all domains in an array.